Gopkg.toml

```diff
@@ -20,3 +20,7 @@
 #  version = "2.4.0"
 
 required = ["github.com/jason0x43/go-toggl"]
+
+[[constraint]]
```

Comment:
Should probably check whether this'll behave the same as the "required" above...?

Reply:
Confirmed all good 👍